tommath.h

```diff
@@ -19,7 +19,7 @@ extern "C" {
 #endif
 
 /* MS Visual C++ doesn't have a 128bit type for words, so fall back to 32bit MPI's (where words are 64bit) */
-#if defined(_MSC_VER) || defined(__LLP64__) || defined(__e2k__) || defined(__LCC__)
+#if (defined(_MSC_VER) || defined(__LLP64__) || defined(__e2k__) || defined(__LCC__)) && !defined(MP_64BIT)
```

Comment:
I wonder if this whole discussion also in #212 missed the point. Isn't __LLP64__ also defined in mingw?

Reply:
> I wonder if this whole discussion also in #212 missed the point. Isn't **LLP64** also defined in mingw?

Maybe, but that doesn't matter for this PR:   This PR makes the default overridable, so by defining -DMP_64BIT we can override the default on any platform.